packages/paste-core/layout/media-object/src/index.tsx

```diff
@@ -1,21 +1,29 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import {Box} from '@twilio-paste/box';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
 import type {BoxProps} from '@twilio-paste/box';
-import {Space, isMarginTokenProp} from '@twilio-paste/style-props';
+import {isMarginTokenProp} from '@twilio-paste/style-props';
+import type {Space} from '@twilio-paste/style-props';
 
-export interface MediaObjectProps extends Pick<BoxProps, 'as' | 'marginTop' | 'marginBottom'> {
+export interface MediaObjectProps
+  extends React.HTMLAttributes<any>,
+    Pick<BoxProps, 'as' | 'element' | 'marginTop' | 'marginBottom' | 'variant'> {
```

Comment:
Is specifying to only use the props listed? I don't think I've used it before and am very intrigued!

Reply:
Yep exactly that. It's the opposite of `Omit`. https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys